src/libsyncengine/syncpal/tmpblacklistmanager.cpp

```diff
@@ -28,7 +28,7 @@ static const int oneHour = 3600;
 namespace KDC {
 
 TmpBlacklistManager::TmpBlacklistManager(std::shared_ptr<SyncPal> syncPal) : _syncPal(syncPal) {
-    LOG_DEBUG(Log::instance()->getLogger(), "TmpBlacklistManager created");
+    LOG_SYNCPAL_DEBUG(Log::instance()->getLogger(), "TmpBlacklistManager created");
```

Comment:
The same modification could be done elsewhere.

Reply:
Thanks, indeed! Following-up with this other PR: https://github.com/Infomaniak/desktop-kDrive/pull/286